app/cdap/components/GridTable/index.tsx

```diff
@@ -142,7 +156,7 @@ export default function GridTable() {
 
   // ------------@createMissingData Function is used for preparing data for second row of Table which shows Missing/Null Value
   const createMissingData = (statistics: IObject) => {
-    const statisticObjectToArray = Object.entries(statistics);
+    const statisticObjectToArray = statistics ? Object.entries(statistics) : [];
```

Comment:
remove Array word from any variable name or function name
if this function converts object to array then rename it to updatedStatisticsData or something appropriate to ur need